WebRoot/css/zm.css

```diff
@@ -3807,6 +3809,37 @@ HTML>BODY .appt-selected .appt_allday_body
 	padding-right: 0;
 }
 
+/* Out-of-Office preference section */
+.ZmOutOfOfficeWrapper {
+	@PrefOOOWrapper@
+}
+
+.ZmOutOfOfficeWrapper textarea {
+	@FullWidth@
+}
+
+.ZmOutOfOfficeWrapper textarea.ZmOOOExternalMsg {
```

Comment:
Yes, we do have two. Other one is hidden by default and shows up when you opt so send custom reply message to external sender.